bokeh/models/scales.py

```diff
@@ -40,7 +41,20 @@ class LogScale(Scale):
     '''
 
     '''
-    pass
+
+    log_type = String(help="""
+    An optional value specifying the type of log. Value values are:
+
+        log (default value)
+        log1p
+
+    """)
+    def __init__(self, *arg, **kw):
+        super(LogScale, self).__init__(*arg, **kw)
+        if "log_type" in kw:
+            self.log_type = kw['log_type']
+        else:
+            self.log_type = 'log'
```

Comment:
All this should be replaced by `log_type = Enum("log", "log1p", default="log", help=...)`.